device_coupler/ovs_helper.py

```diff
@@ -0,0 +1,79 @@
+"""Module to help setup OVS bridges, VxLANs and other virt network components"""
+
+from __future__ import absolute_import, division
+
+from functools import partial
+from python_lib.shell_command_helper import ShellCommandHelper
+from utils import get_logger
+
+
+class OvsHelper:
+    """Class to build OVS, VxLANs and other network components"""
```

Comment:
I don't think "build OVS" is what you mean here -- that implies actually compiling OVS (which is a thing). Do you mean something like construct an OVS bridge?

Reply:
I meant OVS network components but updated it.